typescript/sdk/src/providers/SmartProvider/SmartProvider.ts

```diff
@@ -27,6 +27,16 @@ import {
   SmartProviderOptions,
 } from './types.js';
 
+export const getSmartProviderErrorMessage = (errorMsg: string) =>
+  `RPC request failed with ${errorMsg}. Check RPC validity. To override RPC URLs, see: https://docs.hyperlane.xyz/docs/deploy-hyperlane-troubleshooting#override-rpc-urls`;
```

Comment:
```suggestion
export const getSmartProviderErrorMessage = (errorMsg: string) =>
  `RPC request failed. Check RPC validity. To override RPC URLs, see: https://docs.hyperlane.xyz/docs/deploy-hyperlane-troubleshooting#override-rpc-urls: ${errorMsg}`;
```
or
```suggestion
export const getSmartProviderErrorMessage = (errorMsg: string) =>
  `${errorMsg}: RPC request failed. Check RPC validity. To override RPC URLs, see: https://docs.hyperlane.xyz/docs/deploy-hyperlane-troubleshooting#override-rpc-urls`;
```
to keep the line readable and the logs grepable